web/packages/teleport/src/DeviceTrust/DeviceTrustLocked.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+
+import Box from 'design/Box';
+
+import Text from 'design/Text';
+
+import Flex from 'design/Flex';
+
+import Link from 'design/Link';
+
+import { IconCircle } from 'design/Icon/IconCircle';
+
+import { Windows, Linux, Apple } from 'design/Icon';
+
+import { LockIcon } from 'design/SVGIcon';
+
+import Table, { Cell } from 'design/DataTable';
+
+import {
+  FeatureBox,
+  FeatureHeader,
+  FeatureHeaderTitle,
+} from 'teleport/components/Layout';
+import { CtaEvent } from 'teleport/services/userEvent';
+import { ButtonLockedFeature } from 'teleport/components/ButtonLockedFeature';
+
+export function DeviceTrustLocked() {
+  const theme = useTheme();
+
+  function generateFakeItems(count) {
+    const items = [];
+    const osType = ['Windows', 'Linux', 'macOS'];
+
+    for (let i = 0; i < count; i++) {
+      items.push({
+        id: `id-${i}`,
+        assetTag: `CLFBDAACC${i}`,
+        enrollStatus: `enroll-status-${i}`,
+        osType: osType[i % osType.length],
+      });
+    }
+
+    return items;
+  }
+  return (
+    <FeatureBox>
+      <FeatureHeader>
+        <FeatureHeaderTitle>Trusted Devices</FeatureHeaderTitle>
+      </FeatureHeader>
+      <Box position="relative">
+        <Box
+          width="100%"
+          mr="6"
+          mb="4"
+          style={{
+            filter: 'blur(2px)',
+            pointerEvents: 'none',
+            userSelect: 'none',
+          }}
+        >
+          <FakeDeviceList
+            items={generateFakeItems(15)}
+            fetchData={() => null}
+            fetchStatus={''}
+          />
+        </Box>
+        <StyledMessageContainer>
+          <Box
+            bgColor={theme.colors.spotBackground[0]}
+            p="3"
+            borderRadius="50%"
+          >
+            <IconCircle Icon={LockIcon} size={64} />
+          </Box>
+          <Text fontSize="2" textAlign="center">
```

Comment:
I think the centered text looks awkward here. What say you, @roraback?

Reply:
Adjusted text alignment to `justify` and reduced width to `600px` so that it looks uniform in https://github.com/gravitational/teleport/pull/30249/commits/a6ac29fbff1c4ff851779c6ab67c42c0787e05b1

<img width="1026" alt="Screenshot 2023-08-15 at 10 04 59 AM" src="https://github.com/gravitational/teleport/assets/10177602/bfa77deb-32c4-47f6-8f20-4968535330a7">